src/System.Net.Http/tests/FunctionalTests/DefaultCredentialsTest.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Http.Functional.Tests
     using Configuration = System.Net.Test.Common.Configuration;
 
     // TODO: #2383 - Consolidate the use of the environment variable settings to Common/tests.
-    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "dotnet/corefx #20010")]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "dotnet/corefx #20010")]
```

Comment:
Thi SkipOnTargetFramework could become an ActiveIssue(20010, TargetFrameworkMonikers.Uap) as well now?  Or that can happen in a later PR?